templates/profile.html

```diff
@@ -7,98 +7,105 @@
-        'socials': {},
+      function dataEditTextarea(editable) {
+        editable.innerHTML = `
+          <textarea data-edit
+            name="${editable.dataset.editName}"
+            ${editable.dataset.editAttrs}
+            >${editable.textContent || ''}</textarea>
+        `
       }
-      fields.filter(f => f.name.slice(0, 8) === 'socials-').forEach(input => {
-        data['socials'][input.name.slice(8)] = input.value.trim()
+      Array.from(document.querySelectorAll('[data-edit-name]')).forEach(editable => {
+        const type = editable.dataset.editType || 'input'
+        const funcName = `dataEdit${type[0].toUpperCase()}${type.slice(1)}`
+        this[funcName] ? this[funcName](editable) : dataEditInput(editable)
       })
 
-      request(event.target.action, {
-        method: event.target.dataset.method || event.target.method,
-        body: data,
+      document.querySelector('.user-save').addEventListener('submit', event => {
+        event.preventDefault()
+
+        const fields = Array.from(document.querySelectorAll('[data-edit]'))
+        const data = {
+          'name': fields.find(f => f.name === 'name').value.trim(),
+          'website': fields.find(f => f.name === 'website').value.trim(),
+          'about': fields.find(f => f.name === 'about').value.trim(),
+          'socials': {},
+        }
+        fields.filter(f => f.name.slice(0, 8) === 'socials-').forEach(input => {
+          data['socials'][input.name.slice(8)] = input.value.trim()
+        })
+
+        request(event.target.action, {
+          method: event.target.dataset.method || event.target.method,
+          body: data,
+        })
+        .then(r => location.href = `/profile/${r.id}/`)
+        .catch(e => alert(e.error))
       })
-      .then(r => location.href = `/profile/${r.id}/`)
-      .catch(e => alert(e.error))
-    })
 
-    const deleteForm = document.querySelector('.user-delete')
-    deleteForm && deleteForm.addEventListener('submit', event => {
-      event.preventDefault()
-      const confirmed = confirm('Are you sure you want to delete your account?')
-      if (!confirmed) return
+      const deleteForm = document.querySelector('.user-delete')
+      deleteForm && deleteForm.addEventListener('submit', event => {
+        event.preventDefault()
+        const confirmed = confirm('Are you sure you want to delete your account?')
+        if (!confirmed) return
 
-      request(event.target.action, {method: event.target.dataset.method})
-      .then(r => {
-        console.info('user deleted')
-        location.href = '/'
+        request(event.target.action, {method: event.target.dataset.method})
+        .then(r => {
+          console.info('user deleted')
+          location.href = '/'
+        })
+        .catch(e => alert(e.error))
       })
-      .catch(e => alert(e.error))
-    })
-  </script>
-{% endif %}
+    </script>
+  {% endif %}
+  </section>
+  <section>
```

Comment:
Should this  be `<aside>`?

Reply:
to me `aside` means that is a major column and a minor one : `aside`. Here it does't make sense cause both columns have the same importance. What do you think ?